index.bs

```diff
@@ -690,71 +690,67 @@ See also
 
-The decision about whether a property should be inherited should be based largely
-on what should happen when the property is specified on multiple elements.
 If setting the property on a descendant element needs to override (rather than add to)
 the effect of setting it on an ancestor,
 then the property should probably be inherited.
-If setting the property on a descendant element is a separate effect
-that adds to setting it on an ancestor,
-then the property should probably not be inherited.
-
-One specific example that is common is that if a property has an effect on text,
-then it is almost always in the class where
-a descendant element needs to override (rather than add to) the effect of setting it on an ancestor,
-and the property should be inherited.
-This is also needed to maintain the design principle that
-inserting an unstyled inline element around a piece of text
-does not change the appearance of that text.
-
 A specification of an non-inherited property requiring that the handling of an element
 look at the value of that property on its ancestors (which may also be slow)
 is a "code smell" that suggests that the property likely should have been inherited.
 
-Likewise, a specification of an inherited property requiring that the handling of an element
+If setting the property on a descendant element is a separate effect
+that adds to setting it on an ancestor,
+then the property should probably **not** be inherited.
+A specification of an inherited property requiring that the handling of an element
 ignore the value of a property if it's the same as the value on the parent element
 is a "code smell" that suggests that the property likely should not have been inherited.
 
+<div class="note">
+If a property has an effect on text,
+then it's almost always true that
+a descendant element needs to override (rather than add to)
+the effect of setting it on an ancestor,
+and the property should be inherited.
+This is also needed to maintain the design principle that
+inserting an unstyled inline element around a piece of text
+doesn't change the appearance of that text.
+</div>
+
 <div class="example">
-For example, the 'background-image' property is not inherited.
-It causes an image to be drawn over part or all of the element
-(though this image might not be fully opaque).
-The CSS painting model allows other things to draw on top of that background,
-including the backgrounds of descendant elements.
-But specifying ''background-image: none'' on a descendant element does not
-cause a hole to be poked in the ancestor element's background,
-since the entire effect comes from the property being used on a single element.
-Setting ''background-image: none'' is useful
-to override a declaration lower in the cascade on the same element,
-but this is a different feature from replacing the backgrounds that are drawn by ancestors.
+For example, the 'background-image' property is **not** inherited.
```

Comment:
I'm not sure about dropping the explanation. Even with the above additional text I think this statement could still use an additional sentence of context reiterating the principle.

Reply:
There is still a whole paragraph below connecting it to the principle.